packages/cdk/lib/constants.ts

```diff
@@ -11,6 +11,8 @@ export const USER_EMAIL_TAG_KEY = `${APP_NAME}-user-email`;
 export const AGC_VERSION_KEY = `${APP_NAME}-version`;
 export const VPC_PARAMETER_NAME = "vpc";
 export const VPC_PARAMETER_ID = "VpcId";
+export const VPC_SUBNETS_PARAMETER_NAME = "InfraSubnets";
```

Comment:
It ends up being (part of) an SSM parameter name so it could really be anything.

Reply:
I use 'InfraSubnet' in some places as CDK "id" for some artifacts. The ID is essentially the name of the artifact in CFn. The use of 'InfraSubnet' is in CK really only to avoid any name collisions with "Subnet". So the InfraSubnets in `constants.ts` carries over from that convention but doesn't have to.